Adyen/UI/Form/Items/SelectableFormItem/SelectableFormItemStyle.swift

```diff
@@ -0,0 +1,53 @@
+//
+// Copyright (c) 2024 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+import UIKit
+
+/// Contains the styling customization options for an item in a selectable form.
+public struct SelectableFormItemStyle: ViewStyle {
```

Comment:
I think this is not added to the dropin config's style object?

Reply:
We derive it from the ListComponentStyle